base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -1341,6 +1384,10 @@ impl LMDBDatabase {
             header.height,
             &BlockAccumulatedData::new(kernel_mmr.get_pruned_hash_set()?, total_kernel_sum),
         )?;
+        allow_smt_change.store(false, Ordering::SeqCst);
```

Comment:
When does this get reset?

Reply:
it does not, 
This is to ensure it only ever runs once on smt per transction